### PR TITLE
Example of a Lumen Caddyfile

### PR DESCRIPTION
**Pull request by tobya**

### CODE REVIEW

**Comment by mholt** [comment]
Isn't it normal for Windows paths to use `\` instead of `/`?

**Comment by tobya** [comment]
Updated Now

File: lumen/readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    api.mysite.ie 
+    
+    #rewrite all requests to index.php
+    rewrite / {
+     r (.*)
+     to /index.php
+    }
+    
+    fastcgi / 127.0.0.1:9876 php
+    root "C:\path\tolumen\public"
```
Comment (mholt) [inline lumen/readme.md]: I don't really see the point in duplicating the Caddyfile in the readme. It's okay to have a short readme, and it's less to maintain for me and others. So just take this out and I'll merge it in! Thanks.

**Comment by mholt** [comment]
Thank you!

**Pull request merged**